src/utils.ts

```diff
@@ -67,35 +82,63 @@ function printElement({ props }) {
   )}`;
 }
 
-function display(value) {
+function display(value: any) {
   return typeof value === 'string' ? value : stringify(value);
 }
 
-function getMessage(matcher, expectedLabel, expectedValue, receivedLabel, receivedValue) {
+function getMessage(
+  matcher: string,
+  expectedLabel: string,
+  expectedValue: string | RegExp,
+  receivedLabel: string,
+  receivedValue: string | null,
+) {
   return [
     `${matcher}\n`,
     `${expectedLabel}:\n${expectedColor(redent(display(expectedValue), 2))}`,
     `${receivedLabel}:\n${receivedColor(redent(display(receivedValue), 2))}`,
   ].join('\n');
 }
 
-function matches(textToMatch, matcher) {
+function matches(textToMatch: string, matcher: string | RegExp) {
   if (matcher instanceof RegExp) {
     return matcher.test(textToMatch);
-  } else {
-    return textToMatch.includes(String(matcher));
   }
+
+  return textToMatch.includes(matcher);
 }
 
-function normalize(text) {
+function normalize(text: string) {
   return text.replace(/\s+/g, ' ').trim();
 }
 
-function mergeAll(objects) {
-  return Object.assign({}, ...(objects ?? [{}]));
+type StyleRecord = {
+  [P in keyof TextStyle | keyof ViewStyle]?: TextStyle[P];
+};
```

Comment:
Since this is used in internal only code maybe we can go with `Record<string, unknown>`?

Reply:
To me it define better the type of the record than `Record<string, unknown>` and will help future people who code on the repo, but it will have a cost to maintain. 

I think we should keep it unless it becomes an issue

Reply:
Current type has issues that it does not support `ImageStyle`. 
Would as simple union work?
`type StyleRecord = Partial<ViewStyle> | Partial<TextStyle> | Partial<ImageStyle>` work?

Reply:
End up using the stylesheet.flatten() so this type is removed.